jobs/ci-master.yaml

```diff
@@ -5,7 +5,7 @@
       - git:
           url: https://github.com/charmed-kubernetes/jenkins.git
           branches:
-            - 'master'
+            - 'main'
```

Comment:
I believe this swaps the "branches to build" from master to main.  
Once we merge this PR.  I can run `jjb` from main and update every job.  Is that right @kwmonroe

Reply:
I think so, too.